docs/getting-started/openshift/index.md

```diff
@@ -0,0 +1,187 @@
+# Getting Started - OpenShift Routing
+
+This guide covers how Argo Rollouts integrates with
+[OpenShift Routing](https://docs.openshift.com/enterprise/3.0/architecture/core_concepts/routes.html) 
```

Comment:
Should this be 4.7?

Reply:
I couldn't find an equivalent overview of routes with the 4.7 documentation, this is the closest to an overview/summary of OpenShift Routes